Library/Homebrew/formula_installer.rb

```diff
@@ -398,7 +433,8 @@ def finish
 
     keg = Keg.new(formula.prefix)
     link(keg)
-    fix_install_names(keg)
+
+    fix_install_names(keg) if !pour_bottle? || (pour_bottle? && formula.bottle.needs_relocation?)
```

Comment:
It can be confusing here. Maybe use `unless @poured_bottle && formula.bottle.skip_relocation?`